.github/workflows/deploy.yml

```diff
@@ -8,35 +8,14 @@ on:
     - cron: '0 0 * * *'
 
 jobs:
-  build:
-    strategy:
-      matrix:
-        environment: [iota, shimmer, next]
-    uses: ./.github/workflows/build.reusable.yaml
-    with:
-      environment: ${{ matrix.environment }}
-  config:
-    uses: ./.github/workflows/config.reusable.yaml
   deploy:
     runs-on: ubuntu-latest
-    needs:
-      - build
-      - config
     steps:
       - uses: actions/setup-node@v3
         with:
           node-version: 16
-      # All previous jobs upload to same artifact so only a single download required
-      - name: Download build
-        uses: actions/download-artifact@v3
-        with:
-          name: build
-      # Paths of the build artifacts are preserved (`build`, `build/shimmer`, and `build/next`)
-      - name: Extract build
-        run: |
-          tar -xf iota.tar
-          tar -xf shimmer.tar
-          tar -xf next.tar
+      - name: build
+        uses: ./.github/actions/build/
```

Comment:
I guess same as above, do we need a checkout step so the action can find the referenced composite action?

Reply:
Correct

Reply:
Fixed in b2f6dc5